lib/mayaUsd/ufe/UsdUndoInsertChildCommand.cpp

```diff
@@ -150,16 +154,20 @@ bool UsdUndoInsertChildCommand::insertChildUndo()
     bool status = SdfCopySpec(_parentLayer, _usdDstPath, _childLayer, _usdSrcPath);
     if (status)
     {
+        // we shouldn't rely on UsdSceneItem to access the UsdPrim since 
+        // it could be stale. Instead we should get the USDPrim from the Ufe::Path
+        const auto& srcPrimUsd = ufePathToPrim(_ufeSrcPath);
+
         // remove all scene description for the given path and 
         // its subtree in the current UsdEditTarget
         {
-            auto stage = _ufeDstItem->prim().GetStage();
+            auto stage = srcPrimUsd.GetStage();
```

Comment:
You sure about this?  The original code was getting the stage from the destination item to remove its prim, so I think you should be doing the same.

Reply:
@ppt-adsk  Duh!!  addressed in 9948c423fa9ba43020d6c029f3d008dd53d01df6